### ISSUE TITLE
Copy of Zenhub Access Internal Process Documentation

### ISSUE THREAD

**Issue by alyssagallion** [department-of-veterans-affairs/va.gov-team#77954]
## Description 
The Platform Support Team is taking over the Zenhub access process. We need to document what the process details are.
---

## Definition of Done
All tasks below are complete. 
---

## Tasks 
_Steps to reach definition of done._ 
- [ ] Add the process for adding Zenhub Access [here](https://vfs.atlassian.net/wiki/spaces/PTST/pages/3010396198/Granting+Access+to+Zenhub)
- [ ] Make sure to note that we are only owning access and not IT for Zenhub
- [ ] Add launched item to the [Launched Support Enhancements Confluence Doc](https://vfs.atlassian.net/wiki/spaces/PTST/pages/2632286220/Launched+Support+Enhancements+Documentation)
- [ ] Alyssa to communicate to relevant channels (Alyssa will update which channels in the tasks after the ticket is assigned to her)
--- 

## Resources & Additional Info 
_Links, screenshots, etc. go here_ 

---

**Comment by alyssagallion** [comment]
Waiting on approval of internal process doc from @EWashb to then add this to What We Can Help With page. Pending.

**Comment by alyssagallion** [comment]
Putting on hold until we can complete this ticket: https://app.zenhub.com/workspaces/platform-tech-team-support-634ebbcc35e2bc0015b382fd/issues/gh/department-of-veterans-affairs/va.gov-team/80186

Pages will match either old or new process. Pending.

**Comment by alyssagallion** [comment]
Good to move forward with new process. Putting back into upcoming sprint.

**Comment by alyssagallion** [comment]
Verified that the process is approved by @EWashb  and we have added the announcement of the new process to Team of Teams slide [here](https://docs.google.com/presentation/d/1nAWe6PNHS5O57ryPfvu81i7Mb5T-Uc-h/edit#slide=id.p14).
